urbs.py

```diff
@@ -1894,18 +1895,27 @@ def report(instance, filename, commodities=None, sites=None):
             for sit in sites:
                 created, consumed, stored, imported, exported, derivative = get_timeseries(
                     instance, co, sit)
+                
+                # temporarily drop out the timeseries demand and delta, for a correct overproduction calculation
+                demand = consumed.pop('Demand')
+                demanddelta = consumed.pop('Delta of Demand to shifted Demand')
 
                 overprod = pd.DataFrame(
                     columns=['Overproduction'],
                     data=created.sum(axis=1) - consumed.sum(axis=1) +
                     imported.sum(axis=1) - exported.sum(axis=1) +
                     stored['Retrieved'] - stored['Stored'])
-
+                
+                # add timeseries demand and delta again to consumed
+                consumed = pd.concat(
+                    (consumed, demand.rename('Demand'),
```

Comment:
When you pop, the series has no name. So to set the name, the rename function is used.